sources/engine/Stride.Assets/SpriteFont/Compiler/TrueTypeImporter.cs

```diff
@@ -128,7 +121,7 @@ private unsafe Glyph ImportGlyph(Face face, char character, float fontSize, Font
                 face.LoadGlyph(index, LoadFlags.Render, LoadTarget.Normal);
 
                 var glyphBitmap = face.Glyph.Bitmap;
-                var bufferData = new Span<byte>((byte*)glyphBitmap.Buffer, glyphBitmap.Rows * glyphBitmap.Pitch);
+                var bufferData = new Span<byte>((byte*)glyphBitmap.Buffer, glyphBitmap.Rows * glyphBitmap.Width);
```

Comment:
It makes more sense that the size is `Rows` times `Pitch`. Why that results in an incorrect buffer size? Worst case should be that `Width == Pitch` if byte-sized and no paddings, no?

As `GetFont()` is a new method, if `Pitch` is not correctly returning the pitch, then maybe we should fix it there.

Also related: Why create a new `GetFont()` method returning a `Face` instead of modifying the already existing `GetFontFace()` that returns a `FontFace`?
I ask because maybe we should encapsulate `Face` inside `FontFace` (as it already has a correct `Pitch` property), and not change this line.